HerPublicWebsite.BusinessLogic/HerPublicWebsite.BusinessLogic.csproj

```diff
@@ -1,16 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <LangVersion>12</LangVersion>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="AWSSDK.S3" Version="3.7.104.8" />
-    <PackageReference Include="CabinetOffice.GovUkDesignSystem" Version="1.0.0-456b8e9" />
+    <PackageReference Include="CabinetOffice.GovUkDesignSystem" Version="1.0.0-3f67920" />
     <PackageReference Include="CsvHelper" Version="30.0.1" />
     <PackageReference Include="GovukNotify" Version="6.1.0" />
-    <PackageReference Include="Hangfire.AspNetCore" Version="1.8.0" />
-    <PackageReference Include="Hangfire.Core" Version="1.8.0" />
-    <PackageReference Include="Hangfire.PostgreSql" Version="1.19.12" />
+    <PackageReference Include="Hangfire.AspNetCore" Version="1.8.14" />
+    <PackageReference Include="Hangfire.Core" Version="1.8.14" />
+    <PackageReference Include="Hangfire.PostgreSql" Version="1.20.9" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="Services\EligiblePostcode\EligiblePostcodeData.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
```

Comment:
Do we need both of these? Have you tested without?

Reply:
seems not